### PR TITLE
Edit Text

### PR DESCRIPTION
**Pull request by wataame**

### CODE REVIEW

**Comment by wataame** [comment]
@urakey Please review 🙏

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # itachoco sketch guideline
 
-1. [Naming Rules](#naming-rules)
-1. [Layers](#layers)
-1. [Symbols](#symbols)
-1. [Export](#export)
+1. [Naming Rules](#1-naming-rules)
+1. [Layers](#2-layers)
+1. [Symbols](#3-symbols)
+1. [Export](#4-export)
```
Comment (urakey) [inline README.md:6]: Thanks ♡

File: README.md [hunk 2]
```diff
@@ -85,6 +84,8 @@ Artboard直下とSymbol配下の重なり順は見た目に準じる。（上に
 - Label
 - Form
 
+※ OS標準UIはソート順で一番上にくるように `*` をつける
```
Comment (urakey) [inline README.md:87]: 👍

**Review by urakey — APPROVED** [review]
Looks good to me ✨

**Pull request merged**